go.mod

```diff
@@ -72,12 +72,12 @@ require (
 	go.opentelemetry.io/proto/otlp v1.4.0
 	go.uber.org/goleak v1.3.0
 	go.uber.org/zap v1.27.0
-	golang.org/x/crypto v0.35.0
-	golang.org/x/net v0.33.0
+	golang.org/x/crypto v0.36.0
+	golang.org/x/net v0.38.0
```

Comment:
this will supersede https://github.com/kubernetes/kubernetes/pull/130913, which is fine

Reply:
Oh, it's automatically bumped, because etcd 3.5.21 bumped `golang.org/x/net v0.38.0`